setup.cfg

```diff
@@ -108,6 +110,8 @@ minimal =
 test =
     codecov~=2.0.15
     coverage~=4.5.1
+    dask[array]>=2021.5.0
+    dask[distributed]>=2021.5.0
```

Comment:
I'm 99% sure this is a mistake:
- the `[test]` target is only meant for testing tools.
- most test jobs already include `[full]` dependencies

so either dask is a mandatory dependency, and it should be included in `install_requires`, or it's optional and it should be only under `[full]`